docs/source/index.rst

```diff
@@ -3,9 +3,29 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-PyTorch Lightning
-=================
+Welcome to PyTorch Lightning
+============================
 
+.. raw:: html
+
+    <div class="tutorials-callout-container">
+        <div class="row">
+
+.. Add callout items below this line
+
+.. customcalloutitem::
+   :description: Learn how to leverage the PyTorch Lightning APIs and features for your ML projects with ease in this quickstart guide.
+   :header: Introduction
+   :button_link:  starter/new-project.html
```

Comment:
```suggestion
   :button_link:  starter/introduction.html
```
🔴 you might need to rebase it with master too to avoid failure on master after merge.

Reply:
Updated!